src/main/java/com/countries/database/demo/CountriesApplication.java

```diff
@@ -2,12 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
-@ComponentScan(basePackages = "com.countries.database.demo")
```

Comment:
based on suggestion from Intellij IDEA Ultimate, this annotation is already included in `@SpringBootApplication`.